scene/main/viewport.h

```diff
@@ -591,8 +595,11 @@ class Viewport : public Node {
 	void gui_reset_canvas_sort_index();
 	int gui_get_canvas_sort_index();
 
-	void gui_release_focus();
-	Control *gui_get_focus_owner() const;
+	int gui_get_active_focus_layer() const;
+	void gui_set_active_focus_layer(int p_focus_layer);
+
+	void gui_release_focus(int p_focus_layer);
```

Comment:
Have you thought about providing a default-value `gui_release_focus(int p_focus_layer = 0);`?

Reply:
No reason not to. Will add.